README.md

```diff
@@ -5,3 +5,4 @@ Today, you're going to help create a Student Showcase for you and your classmate
 
 Be sure to follow our modified GitHub workflow for all pull requests. Good luck!
 
+-Joe Asaro
```

Comment:
Please make sure that there is a space between the hyphen and the start of your name

Reply:
wow this resolve is perrrrfect